app/src/main/res/layout/fraction_interaction_item.xml

```diff
@@ -38,8 +39,20 @@
       android:singleLine="true"
       android:text="@={viewModel.answerText}"
       android:textColor="@color/oppiaPrimaryText"
-      android:textColorHint="@color/edit_text_hint"
-      android:textSize="16sp"
-      android:textStyle="italic" />
-  </FrameLayout>
+      android:textColorHint="@color/editTextHint"
+      android:textSize="14sp"
+      android:textStyle="italic"
+      app:textChangedListener="@{viewModel.answerTextWatcher}" />
+
+    <TextView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="16dp"
+      android:layout_marginEnd="8dp"
+      android:minHeight="32dp"
+      android:fontFamily="sans-serif"
+      android:text="@{viewModel.errorMessage.get()}"
```

Comment:
Should we set the visibility of this to invisible unless there's an error message to display?

Reply:
its will be invisible unless there's an error.